docker-compose.yml

```diff
@@ -0,0 +1,99 @@
+mamangcontrol:
+  restart: always
+  build: ../mama-ng-control
```

Comment:
yeah - again, for dev you checkout the repo's as siblings and this allows for that